docs/Operation-Architecture/Orbit_UI/File_Operations/Deleting_Files.rst

```diff
@@ -30,67 +33,57 @@ To delete files from a selected bucket:
 Deleting Versioned Objects
 --------------------------
 
-To completely delete a versioned object, you must issue S3 API commands
-from the command line.
+Deleting versioned objects is difficult because cloud servers are biased towards
+preserving data. While this is useful, it can become problematic when large 
+numbers of objects are under management (during stress testing, for example).
 
-If you have not already done so, follow the instructions at “Zenko from
-the Command Line,” on page 1 to configure one of your nodes to accept
-AWS S3 CLI commands.
+To completely delete a versioned object, you must issue S3 API commands
+from the command line.
 
-From your server node,
+If you have not already done so, follow the instructions at
+:ref:`Zenko from the Command Line` to configure one of your nodes to accept
+AWS S3 CLI commands.
 
-#. List object versions:
+Zenko provides command-line scripts that enable removing versioned objects 
+completely (both removing the object data and its object ID from the namespace).
 
-   ::
+The cleanupBuckets.js script is available in the s3utils pod. 
 
-     $ aws s3api list-object-versions --bucket <bucket> --endpoint-url http://<zenko.endpoint.url> --profile <aws-profile>
+Run it as follows:
 
-   The ``--profile`` option is only required if you have set up more
-   than the default profile. Check this from your Zenko node with
-   ``cat ~/.aws/credentials``. If only the ``[default]`` profile is
-   returned, omit the ``--profile`` option.
+  #. Enable the s3utils pod with::
 
-   The above command returns the object’s version ID and key.
+     $ kubectl run s3utils --image=zenko/s3utils:0.5 -it bash
 
-   ::
+     .. tip::
 
-     {
-       "DeleteMarkers": [
-          {
-             "Owner": {
-                "DisplayName": "user",
-                "ID": "e25bcaf4c8ed41aba50891a671e68d76eddb405bd620252075388c340925c77e"
-             },
-             "IsLatest": true,
-             "VersionId": "39383436323730383139313839343939393939395247303031202032302e30",
-             "Key": "file.txt",
-             "LastModified": "2018-09-18T17:30:08.103Z"
-          }
-       ]
-     }
+	The s3utils pod is disabled by default. You can also enable it 
+	by adding the following to the options file::
 
-#. To delete object versions, use the command:
+	  maintenance:
+	    debug:
+	      enabled: true
 
-    ::
+  #. Exec into the pod with::
 
-       $ aws s3api delete-object --bucket <bucket-name> --version-id <versionID> --key <key> --endpoint-url http://<zenko.endpoint.url> --profile <aws-profile>
+     $ kubectl exec -it s3utils bash
```

Comment:
Adding it via the options file will require the name to be searched for since the name can/will change (similar to looking for the cloud server manager for the InstanceID):
```suggestion
     # First grep for your s3utils pod
     $ kubectl get pods | grep s3utils
     myzenko-zenko-debug-s3utils-7f77f9b5b9-627gz   1/1  Running   0   31m
     # Then exec into the pod
     $ kubectl exec -it myzenko-zenko-debug-s3utils-7f77f9b5b9-627gz bash
```